src/components/shared/chart/Viewport.js

```diff
@@ -24,7 +24,7 @@ const { DOM_DELTA_PAGE, DOM_DELTA_LINE } =
 
```

Comment:
This component has the most complex changes IMO. I don't see any issue in my test but please test also on your side.